arangod/Cluster/TraverserEngine.cpp

```diff
@@ -217,7 +218,7 @@ void BaseEngine::getVertexData(VPackSlice vertex, VPackBuilder& builder,
   builder.openObject();
 
   if (nestedOutput) {
-    builder.add(VPackValue("vertices"));
+    builder.add(VPackValue(VERTICES));
```

Comment:
(minor) We could use: 

```suggestion
    builder.add(VPackValue(StaticStrings::GraphQueryVertices));
```